### PR TITLE
Fix/update package

### LINKED ISSUE #321: bug: npm installation failing
Received bug report that the installation will fail on node v15.11.0 with following message:

```sh
npm ERR! code 128
npm ERR! git dep preparation failed
npm ERR! command /usr/local/Cellar/node/15.11.0/bin/node /usr/local/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/Users/azeemav/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit
npm ERR! npm WARN using --force Recommended protections disabled.
npm ERR! npm ERR! code 128
npm ERR! npm ERR! command failed
npm ERR! npm ERR! command git clone --mirror -q ssh://git@github.com/gulpjs/gulp.git /Users/azeemav/.npm/_cacache/tmp/git-clone-2c870d1c/.git
npm ERR! npm ERR! fatal: destination path '/Users/azeemav/.npm/_cacache/tmp/git-clone-2c870d1c/.git' already exists and is not an empty directory.
npm ERR!
npm ERR! npm ERR! A complete log of this run can be found in:
npm ERR! npm ERR!     /Users/azeemav/.npm/_logs/2021-03-22T09_34_54_550Z-debug.log

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/azeemav/.npm/_logs/2021-03-22T09_35_08_784Z-debug.log
```

image

 and when switched to v12.14.0, it has the following issue:

image

The problem seemed to be fixable with the following suggestion:

https://github.com/ChainSafe/web3.js/issues/3408

by running `npm install github:barrysteyn/node-scrypt#fb60a8d3c158fe115a624b5ffa7480f3a24b03fb`.

### PR DESCRIPTION
**Pull request by SiaChenRen**
fix #321

### CODE REVIEW

File: src/components/UI/Button/Button.test.js [hunk 1]
```diff
@@ -18,7 +18,10 @@ describe("Button component", () => {
 
   it("shows button text", () => {
     const button = renderer.create(<Button>Test Button</Button>);
-    expect(button.toJSON()).toMatchSnapshot();
+    expect(button.toJSON().children[0]).toMatch("Test Button");
+    expect(button.toJSON().props.className).toMatch(
```
Comment (Nebulis) [inline src/components/UI/Button/Button.test.js:22]: let's say we dont care about this :) (to remove)

File: src/components/UI/Input/Input.test.js [hunk 2]
```diff
@@ -16,7 +16,10 @@ describe("Input component", () => {
 
   it("shows Input text", () => {
     const input = renderer.create(<Input>Test Input</Input>);
-    expect(input.toJSON()).toMatchSnapshot();
+    expect(input.toJSON().children[0]).toMatch("Test Input");
+    expect(input.toJSON().props.className).toMatch(
```
Comment (Nebulis) [inline src/components/UI/Input/Input.test.js:20]: let's say we dont care about this :) (to remove)

File: package.json [hunk 3]
```diff
@@ -20,7 +20,8 @@
     "integration:watch": "nodemon --exec \"npm run integration\" --watch src"
   },
   "dependencies": {
-    "@babel/polyfill": "^7.4.0",
+    "@babel/plugin-transform-runtime": "^7.13.10",
```
Comment (Nebulis) [inline package.json:23]: dev does not work:

image

Comment (SiaChenRen) [reply package.json:23]: I have updated the packages

File: src/components/UI/Input/Input.test.js (hunk 2 above)
Comment (SiaChenRen) [reply src/components/UI/Input/Input.test.js]: Removed the test

File: src/components/UI/Button/Button.test.js (hunk 1 above)
Comment (SiaChenRen) [reply src/components/UI/Button/Button.test.js]: Removed the test

**Review by Nebulis — APPROVED** [review]

**Pull request merged**